draft-shafranovich-rfc4180-bis.md

```diff
@@ -70,13 +70,13 @@ changes since the publication of {{!RFC4180}}):
 
 1. Each record is located on a separate line, delimited by a line break (CRLF or LF). For example:
 
-   aaa,bbb,ccc CRLF<br/>
-   zzz,yyy,xxx CRLF
+   aaa,bbb,cccCRLF<br/>
+   zzz,yyy,xxxCRLF
 
-2. The last record in the file may or may not have an ending line break. For example:
+2. The last record in the file must have an ending line break. For example:
```

Comment:
This should be a MUST in capital letters, because of RFC 2119

Reply:
The document contains many lower case "must" statements. Should these be corrected, too?

Reply:
I updated the develop branch with that, forgot to push. It is pushed now - you may want to refresh from origin then update the pull req.